src/index.js

```diff
@@ -1,5 +1,6 @@
 export {
   intlShape,
+  FormattedDate,
```

Comment:
Should we just add all of the components from react-intl?

Reply:
Maybe. I'm going to leave it this way for now.